modules/cnv-configure-liveness-readiness-probes.adoc

```diff
@@ -0,0 +1,177 @@
+`VirtualMachineInstance`, after an initial delay of `120` seconds.
+** The `VirtualMachineInstance` installs and runs a minimal HTTP server
+on port `1500` using `cloud-init`.
++
+----
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-fedora
+  name: vmi-fedora
+spec:
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+    resources:
+      requests:
+        memory: 1024M
+  livenessProbe:
+    initialDelaySeconds: 120
+    periodSeconds: 20
+    tcpSocket:
+      port: 1500
+    timeoutSeconds: 10
+  terminationGracePeriodSeconds: 0
+  volumes:
+  - name: containerdisk
+    registryDisk:
+      image: registry:5000/kubevirt/fedora-cloud-registry-disk-demo:devel
+  - cloudInitNoCloud:
+      userData: |-
+        #cloud-config
+        password: fedora
+        chpasswd: { expire: False }
+        bootcmd:
+          - setenforce 0
+          - dnf install -y nmap-ncat
+          - systemd-run --unit=httpserver nc -klp 1500 -e '/usr/bin/echo -e HTTP/1.1 200 OK\\n\\nHello World!'
+    name: cloudinitdisk
+----
+
+= Define a readiness probe
+
+.Procedure
+
+* Customize a YAML configuration file to create a readiness probe. Readiness
+probes are configured in a similar manner to liveness probes. However, note the
+following differences in this example:
+** Readiness probes are saved using a different spec name. For example, you
+create a readiness probe as `spec.readinessProbe` instead of as
+`spec.livenessProbe.<type-of-probe>`.
+** When creating a readiness probe, you optionally set a `failureThreshold` and a
+`successThreashold` to switch between `ready` and `non-ready` states, should the probe
+succeed or fail multiple times.
++
+----
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-fedora
+  name: vmi-fedora
+spec:
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+    resources:
+      requests:
+        memory: 1024M
+  readinessProbe:
+    httpGet:
+      port: 1500
+    initialDelaySeconds: 120
+    periodSeconds: 20
+    timeoutSeconds: 10
+    failureThreshold: 3
+    successThreshold: 3
+  terminationGracePeriodSeconds: 0
+  volumes:
+  - name: containerdisk
+    registryDisk:
+      image: registry:5000/kubevirt/fedora-cloud-registry-disk-demo:devel
+  - cloudInitNoCloud:
+      userData: |-
+        #cloud-config
+        password: fedora
+        chpasswd: { expire: False }
+        bootcmd:
+          - setenforce 0
+          - dnf install -y nmap-ncat
+          - systemd-run --unit=httpserver nc -klp 1500 -e '/usr/bin/echo -e HTTP/1.1 200 OK\\n\\nHello World!'
+    name: cloudinitdisk
+  ----
```

Comment:
on the rendered page I see `:leveloffset!:` at the end, whatever that means, but I think it doesn't belong there :)

Reply:
Ah, thanks! My end code block (----) was not aligned properly. Done. Will update link for test build shortly.